server/s3_copy_object_action.h

```diff
@@ -57,6 +58,12 @@ class S3CopyObjectAction : public S3ObjectAction {
   unsigned short tried_count;
   int layout_id;
   int old_layout_id;
+  int motr_unit_size;
+  unsigned motr_write_payload_size;
+  bool copy_failed = false;
+
+  size_t content_length;
+  size_t rest_bytes;
```

Comment:
should we re-name this to "bytes_left_to_read" or "bytes_still_to_read" ?

Reply:
'content_length' contains size of the object. It's set once and isn't changed futher.
Will rename 'rest_bytes'.